docs/evaluation/how_to_guides/evaluation/use_prebuilt_llm_evaluator.mdx

```diff
@@ -0,0 +1,128 @@
+---
+sidebar_position: 14
+---
+
+# Use Prebuilt LLMEvaluator (Python only)
+
+:::tip Recommended Reading
+Before diving into this content, it might be helpful to read the following:
+
+- [LangChain evaluator reference](/reference/sdk_reference/langchain_evaluators)
+
+:::
+
+When working with applications that produce unstructured text outputs, it can be challenging to assess the quality and accuracy of those outputs without involving a human expert or another LLM.
```

Comment:
How to guides are meant to instruct users rather than explain: https://www.diataxis.fr/how-to-guides/

Maybe replace this whole thing with:


> LangSmith provides a collection of prebuilt LLM-as-judge evaluators that you can use to assess the quality of your application across a host of common categories (hallucination, conciseness, bias, etc). These are designed to both work off the shelf by default, and be customizable to your specific needs.